File: C#/z2_konwerter.cs
using System;

class Program
{
    static void Main()
    {
        Console.WriteLine("Konwerter temperatur");
        Console.WriteLine("C - Celsjusz -> Fahrenheit");
        Console.WriteLine("F - Fahrenheit -> Celsjusz");

        Console.Write("Wybierz opcję (C/F): ");
        string wybor = Console.ReadLine();

        if (wybor == "C")
        {
            Console.Write("Podaj temperaturę w stopniach Celsjusza: ");
            double c = double.Parse(Console.ReadLine());

            double f = c * 1.8 + 32;

            Console.WriteLine(c + "°C = " + f + "°F");
        }
        else if (wybor == "F")
        {
            Console.Write("Podaj temperaturę w stopniach Fahrenheita: ");
            double f = double.Parse(Console.ReadLine());

            double c = (f - 32) / 1.8;

            Console.WriteLine(f + "°F = " + c + "°C");
        }
        else
        {
            Console.WriteLine("Blędna opcja");
        }
    }
}

File: python/z2_konwerter.py
print("Konwerter temperatur")
print("C - Celsjusz -> Fahrenheit")
print("F - Fahrenheit -> Celsjusz")

wybor = input("Wybierz opcję (C/F): ")

if wybor == 'C':
    c = float(input("Podaj temperaturę w stopniach Celsjusza: "))
    f = c * 1.8 + 32
    print(f"{c} °C = {f} °F")

elif wybor == 'F':
    f = float(input("Podaj temperaturę w stopniach Fahrenheita: "))
    c = (f - 32) / 1.8
    print(f"{f} °F = {c} °C")

else:
    print("Błędna opcja")
